auth0/src/main/java/com/auth0/android/request/internal/RequestFactory.java

```diff
@@ -126,7 +131,11 @@ public AuthenticationRequest authenticationPOST(HttpUrl url, OkHttpClient client
     }
 
     <T, U extends Auth0Exception> ParameterizableRequest<T, U> createSimpleRequest(HttpUrl url, OkHttpClient client, Gson gson, String method, Class<T> clazz, ErrorBuilder<U> errorBuilder) {
-        return new SimpleRequest<>(url, client, gson, method, clazz, errorBuilder);
+        SimpleRequest<T, U> request = new SimpleRequest<>(url, client, gson, method, clazz, errorBuilder);
+        if ("POST".equals(method) && Credentials.class.equals(clazz)) {
+            request.setJwtVerifier(verifier);
```

Comment:
If this is an "Authentication request", set the verifier. Internally, verification will only happen when an Id Token is present on the Credentials response.